.github/ISSUE_TEMPLATE/2_bug.md

```diff
@@ -16,36 +16,72 @@ assignees: ''
       * https://api.flutter.dev/
       * https://stackoverflow.com/questions/tagged/flutter?sort=frequent
 
+     Issues directly asking for support will be closed.
+
      If you have found a bug or if our documentation doesn't have an answer
      to what you're looking for, then fill out the template below. Please read
      our guide to filing a bug first: https://flutter.dev/docs/resources/bug-reports
 -->
 
-## Steps to Reproduce
+## Description
```

Comment:
i picked a "description" over "details" here because this template because it just better suits imo

Reply:
If we use "Details", most people would end up describing the nature of the issue rather than providing necessary steps to replicate the issue.

Reply:
steps to reproduce is part of "details"/"description" section (see below)

Reply:
literally the only thing we're asking for here is the steps to reproduce though, no?

Reply:
in this section i mean